shared/AppInsightsCore/src/JavaScriptSDK/StatsBeat.ts

```diff
@@ -0,0 +1,182 @@
+                    return;
+                }
+                if (payloadData && payloadData["statsBeatData"] && payloadData["statsBeatData"]["startTime"]) {
+                    _networkCounter.totalRequest++;
+                    _networkCounter.requestDuration += utcNow() - payloadData["statsBeatData"]["startTime"];
+                }
+                let retryArray = [401, 403, 408, 429, 500, 503];
+                let throttleArray = [402, 439];
+                if (status === 200) {
+                    _networkCounter.success++;
+                } else if (retryArray.includes(status)) {
+                    _networkCounter.retry[status] = (_networkCounter.retry[status] || 0) + 1;
+                } else if (throttleArray.includes(status)) {
+                    _networkCounter.throttle++;
+                } else if (status !== 307 && status !== 308) {
+                    _networkCounter.failure[status] = (_networkCounter.failure[status] || 0) + 1;
+                }
+                _setupTimer();
+            };
+
+            
+            
+            _self.countException = (endpoint: string, exceptionType: string) => {
+                if (!_isEnabled || !_checkEndpoint(endpoint)) {
+                    return;
+                }
+                _networkCounter.exception[exceptionType] = (_networkCounter.exception[exceptionType] || 0) + 1;
+                _setupTimer();
+            }
+
+            function _setupTimer() {
+                if (!_timeoutHandle) {
+                    _timeoutHandle = scheduleTimeout(() => {
+                        _timeoutHandle = null;
+                        this.trackShortIntervalStatsbeats();
+                    }, STATS_COLLECTION_SHORT_INTERVAL);
+                }
+            }
+
+            _self.trackShortIntervalStatsbeats = (): void => {
+                _trackSendRequestDuration();
+                _trackSendRequestsCount();
+                _networkCounter = createNetworkStatsbeat(_networkCounter.host);
+                _timeoutHandle && _timeoutHandle.cancel();
+                _timeoutHandle = null;
+            }
+
+            function _checkEndpoint(endpoint: string) {
+                return _networkCounter && _networkCounter.host === endpoint;
+            }
+
+            function _getCustomProperties(ikey: string) {
+                _cikey = ikey;
+                _language = STATSBEAT_LANGUAGE;
+                _os = STATSBEAT_TYPE;
+                _runTimeVersion = STATSBEAT_TYPE;
+            }
+
+            function _sendStatsbeats(name: string, val: number, properties?: {}) {
+                // Add extra properties
+                let baseProperties = {
+                    "rp": "unknown",
+                    "attach": "Manual",
+                    "cikey": _cikey,
+                    "runtimeVersion": _runTimeVersion,
+                    "os": _os,
+                    "language": _language,
+                    "version": _sdkVersion,
```

Comment:
For the other sdks, we do not populate this with `sdkVersion` but rather JUST the version of the SDK.

Reply:
@siyuniu-ms 

I am not sure why this is being resolved? Seems like you are still using `sdkVersion` instead of just `version`.